src/GRA.Web/Views/Events/Detail.cshtml

```diff
@@ -1,5 +1,7 @@
 ﻿@model GRA.Controllers.ViewModel.Events.EventsDetailViewModel
 
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
SharedLocalizer doesn't need to be injected here as it's coming in through the _ViewImports.

Reply:
Fixed in commit 97b7d4757c5daf38b97e01a4b0f363bb290ee169